.gitignore

```diff
@@ -5,6 +5,9 @@ tigerbeetle
 zig/
 zig-cache/
 
+# IDEs
+.idea
+
```

Comment:
This would be better left out, to be added into a global `.gitignore` that's referenced by your `.gitconfig`. Otherwise, we'll end up with a ton of workspace-specific ignores here.